project-metrics/metrics_service/.gcloudignore

```diff
@@ -15,5 +15,9 @@
 
 # Python pycache:
 __pycache__/
+.pytest_cache/
+
 # Ignored by the build system
-/setup.cfg
\ No newline at end of file
+/setup.cfg
+
+env.example.yaml
```

Comment:
Extrasupernit: newlines at end of files 🙃

Reply:
:+1: